synapse/server_notices/resource_limits_server_notices.py

```diff
@@ -91,24 +92,28 @@ def maybe_send_server_notice_to_user(self, user_id):
         currently_blocked, ref_events = yield self._is_room_currently_blocked(room_id)
 
         try:
-            if not self._config.mau_limit_alerting:
-                # Alerting disabled, reset room if necessary and return
+            is_auth_blocking = False
+            event_limit_type = ""
+
+            try:
+                # Normally should always pass in user_id to check_auth_blocking
+                # if you have it, but in this case are checking what would happen
+                # to other users if they were to arrive.
+                yield self._auth.check_auth_blocking()
+            except ResourceLimitError as e:
+                is_auth_blocking = True
+                event_body = e.msg
+                event_limit_type = e.limit_type
+
+            if (
+                not self._config.mau_limit_alerting
+                and event_limit_type is LimitBlockingTypes.MONTHLY_ACTIVE_USER
```

Comment:
use `==` not `is` except when checking for None

```suggestion
                and event_limit_type == LimitBlockingTypes.MONTHLY_ACTIVE_USER
```